freqtrade/templates/base_strategy.py.j2

```diff
@@ -78,8 +78,8 @@ class {{ strategy }}(IStrategy):
 
     # Optional order time in force.
     order_time_in_force = {
-        'buy': 'gtc',
-        'sell': 'gtc'
+        'enter': 'gtc',
+        'exit': 'gtc'
```

Comment:
please do the "parameter" renaming in a seperate PR.
That makes it clear which parts had to change because of the strategy interface (column) changes, and which because of this.